CONTRIBUTING.md

```diff
@@ -49,32 +56,46 @@ Subscribe to all the MQTT topics the `yggd` client will publish and subscribe to
 to monitor the MQTT traffic.
 
 ```
-sub -broker tcp://test.mosquitto.org:8883 -topic yggdrasil/$(hostname)/#
+sub -broker tcp://localhost:1883 -topic yggdrasil/$(hostname)/#
 ```
 
 ### Terminal 4
 
 Publish a "PING" command to the `yggd` "control/in" topic.
 
 ```
-go run ./cmd/yggctl generate control-message --type command '{"command":"ping"}' | pub -broker tcp://test.mosquitto.org:8883 -topic yggdrasil/$(hostname)/control/in
+go run ./cmd/yggctl generate control-message --type command '{"command":"ping"}' | \
+    pub -broker tcp://localhost:1883 -topic yggdrasil/$(hostname)/control/in
 ```
 
 You should see the message logged to the output of `sub` in [Terminal
 3](#terminal-3) and receipt of the message logged in the output of `yggd` in
-[Terminal 1](#terminal-1).
+[Terminal 1](#terminal-1). The `echo` worker does not receive any message in this
+case.
 
-Now publish a data message to the echo worker.
+Now publish a data message "hello" to the `echo` worker.
 
 ```
-go run ./cmd/yggctl generate data-message --directive worker "hello" | pub -broker tcp://test.mosquitto.org:8883 -topic yggdrasil/$(hostname)/data/in
+go run ./cmd/yggctl generate data-message --directive echo "hello" | \
+    pub -broker tcp://localhost:1883 -topic yggdrasil/$(hostname)/data/in
 ```
 
+> Note: if you implemented your own worker (e.g. called cool_worker),
+> you will need to replace the `echo` directive with the name of your
+> worker (--directive cool_worker).
+
 Again, you should see the message logged by the `sub` command in [Terminal
 3](#terminal-3), the receipt of the message logged in the output of `yggd` in
 [Terminal 1](#terminal-1). This time, you should see output in [Terminal
 2](#terminal-2) when the echo worker receives the message.
 
+There is another option how to test the echo worker. You can use the `yggctl`
+like this:
+
+```bash
+echo -n "hello" | go run ./cmd/yggctl dispatch -w "echo" -
+```
+
```

Comment:
I'd like to keep the quickstart as short and simple as possible. If we don't have a section below on `yggctl`'s capabilities, then let's add one. Otherwise, can we move this to that section?

Reply:
OK, I will move it to that section.